assignments/02-airbnb-layout/index.html

```diff
@@ -7,6 +7,31 @@
   <title>Hello Front-End</title>
 </head>
 <body>
-  <h1>Hello Front-End</h1>
+  <h1>Meet Guide Books</h1>
+  <p>Discover hunderds of local spots recommended dy Airbnb hosts</p>
+  
+  <div class="city">
+  <div class="cities">
+     <p>San Francisco</p>
+  </div>
+  <div class="cities">
+    <p>New York</p>
+  </div>   
+  <div class="cities">
+    <p>london</p>
+  </div> 
+  </div>
+  <button>See All Guide books</button>
+  <h1>Just for the weekend</h1>
+  <p>Discover new, inspiring places close to home.</p>
+
+  <div class="city"> 
+    <p class="cities">Napa</p>
```

Comment:
It's better use `div` rather than `p` (like what you did above 'line 14'), because `p` defines a paragraph.